plugins/scaffolder/package.json

```diff
@@ -58,6 +58,7 @@
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^10.4.1",
     "@testing-library/user-event": "^12.0.7",
+    "@testing-library/react-hooks": "^3.3.0",
```

Comment:
What I would like to see in the description of your PR:
* what you have added in that is new
* what is the same as the catalog filter
* what applies _just_ to the scaffolder

it will make it easier to review.

Reply:
Updated description